rcl/test/rcl/test_graph.cpp

```diff
@@ -879,7 +879,6 @@ void expect_topics_types(
   rcl_names_and_types_t nat{};
   nat = rcl_get_zero_initialized_names_and_types();
   ret = func(node, topic_name, &nat);
-  ASSERT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
```

Comment:
i think keeping this assertion makes sense to check if any error happens. probably what we really want to do here is avoiding `RMW_RET_NODE_NAME_NON_EXISTENT` case only? that said, if `RMW_RET_NODE_NAME_NON_EXISTENT` returns, it will return immediately to next attempt. and if the error is anything else, assert here to stop the test?

Reply:
To return immediately will change the behavior of the `expect` and the debug log. I'll revert to https://github.com/ros2/rcl/pull/1063/commits/d0d9ba336ef8be6d64632af9d93b95c855cce62c.